mkdocs.yml

```diff
@@ -25,6 +25,7 @@ nav:
     - How do I ...?: filtering/howdoi.md
     - Order of Filtering Operations: filtering/order.md
     - Descaling: filtering/descaling.md
+    - Dehardsubbing: filtering/dehardsubbing.md
```

Comment:
I put this at the bottom since this sort of filtering rarely comes up in modern stuff, but if we wanna keep it in the ideal order of operations I can move it up above descaling.